src/main/java/frc/robot/RobotContainer.java

```diff
@@ -74,7 +75,7 @@ public class RobotContainer {
 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
-      SmartDashboard.putData("autoBalance", swerveSubsystem.autoBalance());
+    LoggingWrapper.shared.add("autoBalance", swerveSubsystem.autoBalance());
```

Comment:
Can we wrap commands in the logger without it breaking?

Reply:
Actually now that i think about it its probably fine, itll just tell is when its running.